_posts/2016-06-17-jess-intro.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Jess Hamilton Introduction"
+date:   2016-06-24
+author: Jess Hamilton
+---
+
+![An image]({{ site.url }}/assets/images/hamilton.jpg)
```

Comment:
You might want to copy the format used here: 

https://github.com/uwescience/DSSG2016/blame/gh-pages/_posts/2016-06-24-introducing-dssg-fellow-kara-woo.md#L8-L9

(setting the image width to be appropriate for your file). In particular, if you remove the line break between the insertion of the image and the next paragaph, the front page will contain both your photo and the first paragraph of your post, which looks slightly better than having just your photo.